3p/ampcontext.js

```diff
@@ -100,7 +100,7 @@ export class AbstractAmpContext {
     /** @protected {!IframeMessagingClient} */
     this.client_ = new IframeMessagingClient(win);
     this.client_.setHostWindow(this.getHostWindow_());
-    this.client_.setSentinel(dev().assertString(this.sentinel));
```

Comment:
currently, `user()` does not notify anyone either. i think probably safe to do dev() for now and take actions with real cases.